tests/jupyterhub_api/test_webReachable.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
Does it make sense to move this test to the web//http category (which is still to be implemented)?
In the end, you try to open a connection and see if the service replies with an 'OK' answer or some error is returned.